src/Fantomas.Core/FormatConfig.fs

```diff
@@ -25,6 +25,24 @@ type MultilineFormatterType =
         | "number_of_items" -> Some(box MultilineFormatterType.NumberOfItems)
         | _ -> None
 
+type MultilineBracketStyle =
+    | Classic
```

Comment:
We probably want to go with something other else for `Classic`.
I think Don call this `Cramped` style. `Pico` is another [option I guess](https://en.wikipedia.org/wiki/Indentation_style#Pico_style).

Reply:
I like cramped!